.circleci/config.yml

```diff
@@ -227,21 +228,21 @@ workflows:
           filters:
             branches:
               only:
-                - release/*
+                - /^release\/.*/
```

Comment:
@dylankelly just and FYI, it's not actually running the full test suites against release branches as the regex was not quite right.